docs/src/components/Header/SkipToContent.astro

```diff
@@ -0,0 +1,21 @@
+<style>
+.skiplink,
+.skiplink:focus,
+.skiplink:focus-visible {
+  position: absolute;
+  padding: 0.25em;
+  font-size: larger;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 9;
+  display: block;
+  text-align: center;
+  background-color: var(--theme-text-accent);
+  color: var(--theme-bg);
+  border-radius: 0.25em;
+  outline: var(--theme-bg) solid 1px;
+  outline-offset: 0;
+}
+</style>
+<a href="#article" class="sr-only focus:not-sr-only skiplink"><span>Skip to Content</span></a>
```

Comment:
i'm not sure whether "focus:not-sr-only" exists in the project 🤔 i don't remember whether I ended up adding it..

Reply:
nope, I don't see it. removed!